src/builtins.ts

```diff
@@ -113,11 +114,24 @@ function any(x: collections.Iterable): boolean {
 }
 
 function bool(x) {
-    if (x === undefined)
+    
```

Comment:
Ya, this appears to be a cleaner implementation. I'll modify the code and submit a new PR.